README.md

```diff
@@ -49,13 +43,12 @@ Configuring Your AndroidManifest
     </application>
     ```
 
--   Enable the `INTERNET`, `ACCESS_NETWORK_STATE` and `GET_TASKS` permissions:
+-   Optional: Enable the `ACCESS_NETWORK_STATE` and `GET_TASKS` permissions:
```

Comment:
I'd rather say "recommended" here, since having these available massively aids in debugging. We can possibly remove `GET_TASKS` since we get this information by default now in Android 5.0+.